ws-nextjs-app/pages/[service]/send/[id]/FormContext/utils/validateFunctions.ts

```diff
@@ -108,15 +108,74 @@ const isValidTel: (data: FieldData) => FieldData = (data: FieldData) => {
   };
 };
 
-const isValidFile: (data: FieldData) => FieldData = (data: FieldData) => {
-  const { value, required, wasInvalid } = data;
+const isValidFiles: (data: FieldData) => FieldData = (data: FieldData) => {
+  const { value, required, wasInvalid, min, fileTypes, max } = data;
```

Comment:
Could we rename `value` here as it's not too clear that it's an array of files when you look at the `validatedFiles` map(), as it uses `file` and `value`, which was a bit confusing at first